### PR TITLE
Hardening/117 post subscribe context conv op

### PR DESCRIPTION
**Pull request by kzangeli**
POST /v1/contextSubscriptions now has its very own service routine, supporting a few URI parameters.

Point of discussion: if URI param 'entity::type=XXX' is used, then the all entity types in the entity-vector are overwritten with this 'XXX' ...

As I said, let's discuss how we want this to work.

### CODE REVIEW

File: src/lib/serviceRoutines/postSubscribeContext.cpp [hunk 1]
```diff
@@ -36,6 +36,15 @@
...
 * postSubscribeContext - 
+*
+* POST /v1/subscribeContext
+* POST /ngsi10/subscribeContext
+*
+* Payload In:  SubscribeContextRequest
+* Payload Out: SubscribeContextResponse
+*
+* URI parameters
+*   - notifyFormat=XXX    (handled by mongoBackend)
```
Comment (fgalan) [inline src/lib/serviceRoutines/postSubscribeContext.cpp]: We dedided to ignore entity::type... I guess that exist and !exist are also ignored, aren't they?

Comment (fgalan) [inline src/lib/serviceRoutines/postSubscribeContext.cpp]: Ah... I have just realize that this is a std-op, so all the URI params for filters are ignored. NTC

File: test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextConvOp.test [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    }
+  ]
+}
+=======================================
+
+
+--TEARDOWN--
+brokerStop CB
+accumulatorStop
+#dbDrop CB
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextConvOp.test]: Uncomment this?

File: src/lib/serviceRoutines/postSubscribeContext.cpp (hunk 1 above)
Comment (kzangeli) [inline src/lib/serviceRoutines/postSubscribeContext.cpp]: Just to clarify, in the other service routine (postSubscribeContextConvOp) we noi longer treat 'entity::type=XXX', but we do treat 'exist=entity::type' and '!exist=entity::type'.
And I just realized I have no func-test for that :-( :-D

File: test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextConvOp.test (hunk 2 above)
Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextConvOp.test]: of course!

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/postSubscribeContextConvOp.test]: Fixed in aedc081

**Comment by fgalan** [comment]
LGTM

**Pull request merged**